src/app/common/data-card/DataCard.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <KCard
+    class="data-card"
+  >
+    <template #body>
+      <dl>
+        <div class="card">
+          <dt class="title">
+            <slot name="title" />
+          </dt>
+          <dd class="body">
+            <slot name="default" />
+          </dd>
+        </div>
+      </dl>
+    </template>
+  </KCard>
+</template>
+<style lang="scss" scoped>
+.data-card {
+  padding: $kui-space-50;
+}
+.card,
+.title,
+.body > :deep(dl),
+.body > :deep(dl) > div {
+  display: flex;
+  gap: $kui-space-40;
+  align-items: flex-start;
+}
+.card {
+  flex-direction: column;
+}
+.title {
+  /* FIXME(jc): Usually this is 1.5 but that looks way to large */
+  line-height: 1.2;
```

Comment:
@kleinfreund do you have any input here? I looked a little deeper into it which resulted in https://github.com/kumahq/kuma-gui/pull/1816 (which is in draft as it still doesn't feel quite right to me I'm gonna check in with @adamdehaven before I go any further with that draft PR)

I do think everything should be 1.3 though, I'll make some more comment on that draft PR.

Reply:
Really you should/could be using the SCSS variables from `@kong/design-tokens`

If you do, the `kui-font-size` and `kui-line-height` variables are meant to be used 1:1, meaning if you have `$kui-font-size-40` you would typically use `$kui-line-height-40` alongside it.

Reply:
Ah you're around! I wanted to chat to you about this along with the PR mentioned above, I know its not ideal. Umm let me think about what you said here a bit in the context of #1816 it might help with that.

Umm if you are only "around" but not "around" (😆 ) we can catch up early next week 👍  (otherwise ping me on Slack)

Oh also see the other FIXME here on the colors